sdk-python/littlehorse/worker.py

```diff
@@ -333,7 +334,7 @@ async def _report_task(self, task_result: ReportTaskRun, retries_left: int) -> N
         )
 
         try:
-            await self._stub.ReportTask(task_result)
+            await self._stub.ReportTask(task_result, timeout=GRPC_UNARY_CALL_TIMEOUT)
```

Comment:
Oh, I see. It's used here. Some comments:

1. Default deadline for LittleHorse grpc requests are 60 seconds
2. It should be configurable (probably not in this PR though)